layouts/layouts/src/main/java/com/microsoft/device/dualscreen/layouts/ViewExtensions.kt

```diff
@@ -14,4 +16,8 @@ val View.locationOnScreen: Point
         val location = IntArray(2)
         getLocationOnScreen(location)
         return Point(location[0], location[1])
-    }
\ No newline at end of file
+    }
+
+fun ViewGroup.hasChild(@IdRes childId: Int): Boolean {
+    return findViewById<View>(childId) != null
```

Comment:
You could also have here

`fun ViewGroup.hasChild(@IdRes childId: Int): Boolean = findViewById<View>(childId) != null`

Reply:
Done